Sources/WebAuthn/Helpers/KeyedDecodingContainer+decodeURLEncoded.swift

```diff
@@ -13,7 +13,7 @@
 
 import Foundation
 
-extension KeyedDecodingContainer {
+public extension KeyedDecodingContainer {
```

Comment:
Can we have the public declarations on the functions to be explicit rather than just on the extension itself